java/src/jmri/implementation/TwoIndexTcsProgrammerFacade.java

```diff
@@ -101,12 +105,20 @@ synchronized public void writeCV(String CV, int val, jmri.ProgListener p) throws
 
     @Override
     synchronized public void readCV(String CV, jmri.ProgListener p) throws jmri.ProgrammerException {
+       readCV(CV, p, 0);
+    }
+
+    @Override
+    synchronized public void readCV(String CV, jmri.ProgListener p, int startVal) throws jmri.ProgrammerException {
         useProgrammer(p);
         parseCV(CV);
+        _startVal = startVal;
+        _startMSB = startVal/256;
+        _startLSB = startVal%256;
```

Comment:
No need to change, but I personally thinks the code is easier to read if there is a space between the operators.
```
_startMSB = startVal / 256;
_startLSB = startVal % 256;
```